eslint-config-fusion-flowtype/.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+<!--
```

Comment:
Will this will work within a nested dir of a monorepo?

Reply:
Just noticed we have these in other packages. We can leave this I guess, but will probably remove these from all packages eventually